src/pages/Background/index.js

```diff
@@ -1,2 +1,66 @@
+//Even while development.. extension need to be reloaded for changes here to come into effect
+import { closest, distance } from 'fastest-levenshtein';
+
 console.log('This is the background page.');
+
 console.log('Put the background scripts here.');
+
+console.log('new line by hamza');
+
+//fetch('http://localhost:5000').then((res) => res.json()).then((res) => { console.log(JSON.stringify(res)) })
+
+// chrome.runtime.onInstalled(() => {
+//     chrome.runtime.onMessage.addListener((msg, sender, res) => {
+//         alert(msg)
+//     })
+// })
+
+let symbolData;
+
+fetch('https://script.google.com/macros/s/AKfycbxJPjCcowBQS124lLZfmw1ItprLtxNx7MiAmTgsdcp_dpTazJJVM7zEOrM_KHPSoib4/exec', {
+    method: 'GET',
+    mode: 'cors'
+}).then(res => res.json()).then((data) => {
+    console.log(data)
+    symbolData = data
+})
+
+chrome.runtime.onMessage.addListener((message, sender, sendRes) => {
+    if (!message.msg) return
+    onClickHandler(message.msg)
+})
+
+function onClickHandler(clickedSymbol) {
```

Comment:
what onclick handler? needs to be more verbose on what this function is doing... could be `onSymbolClick`